test/e2e/md_remediations_test.go

```diff
@@ -21,7 +21,14 @@ var _ = Describe("When testing MachineDeployment remediation [healthcheck] [reme
 		Expect(osType).ToNot(Equal(""))
 		validateGlobals(specName)
 		// We need to override clusterctl apply log folder to avoid getting our credentials exposed.
-		clusterctlLogFolder = filepath.Join(os.TempDir(), "clusters", bootstrapClusterProxy.GetName())
+		clusterctlLogFolder = filepath.Join(os.TempDir(), "target_cluster_logs", bootstrapClusterProxy.GetName())
+
+		Logf("remove existing bmh from source")
+		cmd := exec.Command("bash", "-c", "kubectl delete -f bmhosts_crs.yaml  -n metal3")
```

Comment:
Should we use https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/test/e2e/common.go#L948 instead?

Reply:
done